lib/form_data.js

```diff
@@ -193,7 +193,8 @@ FormData.prototype._getContentDisposition = function(value, options) {
 
   // custom filename takes precedence
   // fs- and request- streams have path property
-  var filename = options.filename || value.path;
+  // formidable and the browser add a name property.
+  var filename = options.filename || value.name || value.path;
```

Comment:
My main concern that `value.name` could be many different things
and we might end up with bunch of false positives.

Can you provide separate integration test instead of augmenting existing custom filename test,
it would better illustrate the use case and would allow for better understanding why we have that piece of code six months from now :)